clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/TerminateGoogleInstancesDescription.groovy

```diff
@@ -16,7 +16,9 @@
 
 package com.netflix.spinnaker.clouddriver.google.deploy.description
 
-class TerminateGoogleInstancesDescription extends AbstractGoogleCredentialsDescription {
+import com.netflix.spinnaker.clouddriver.security.resources.ServerGroupNameable
+
+class TerminateGoogleInstancesDescription extends AbstractGoogleCredentialsDescription implements ServerGroupNameable {
   String serverGroupName
   List<String> instanceIds
```

Comment:
I don't know if you're checking the actual operation itself that the instanceIds belong to the specified server group ... in aws-land we weren't and pretty sure ended up fetching the instanceIds and grabbing the server group names off of them.

Reply:
I believe GCE instance IDs are unique numbers, and thus the only thing I _could_check is the name.